lambdas/url_access_patterns/lambda_for_getHash/index.js

```diff
@@ -0,0 +1,34 @@
+const AWS = require("aws-sdk");
+
+//AWS.config.loadFromPath("../../../keys.json");
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: message,
+  };
+}
+
+exports.getHash = async (event) =>{
+    try{
+        var params={
+            TableName:"V-Transfer",
+            IndexName:"FIND_FILE_BY_URLID",
+            KeyConditionExpression: "#PK= :pk",
+            ExpressionAttributeNames:{
```

Comment:
can we remove this and define key in place?

Reply:
Do you mean one external template file for all lambda functions?

Reply:
no, I mean using `KeyConditionExpression: "PK= :pk",`, and removing the 
```
ExpressionAttributeNames:{
    "#PK": "GS1_PK",            
}
```

Reply:
yup we can do that